scroll/src/component/Chat/Chat.js

```diff
@@ -1,40 +1,76 @@
 import React, { useEffect, useState } from "react";
-import { user } from "../Join/Join";
+// import { user } from "../userJoin/Join.js";
+import { user } from "../userJoin/join";
 import socketIo from "socket.io-client";
-
+let socket;
 const ENDPOINT = "http://localhost:4500/";
```

Comment:
It's a good practice to store your endpoint URL in an environment variable. Create a .env file in the root of your project if you don't already have one. In the .env file, define your  URL as an environment variable.you can access this environment variable using process.env